docs/src/architecture/accounts.md

```diff
@@ -1,27 +1,44 @@
 # Accounts
-An account is an entity which holds assets and defines rules of how these assets can be transferred. The diagram below illustrates basic components of an account.
+An account is an entity which holds assets and defines rules of how these assets can be transferred. The diagram below illustrates basic components of an account. In Miden every Account is a smart contract.
 
 <p align="center">
-    <img src="../diagrams/protocol/account/Account_Definition.png">
+    <img src="../diagrams/architecture/account/Account_Definition.png">
```

Comment:
Maybe we should add `nonce` in the picture as well? (since that's another component of an account).

Reply:
done